timemachine/cpp/src/gpu_utils.cu

```diff
@@ -1,5 +1,13 @@
 #include "gpu_utils.cuh"
 
+int round_up_even(int count) {
+    if (count % 2 == 1) {
+        return count += 1;
+    } else {
+        return count;
+    }
```

Comment:
nit: simplify (also I think `+=` is a typo?)
```suggestion
    return count + (count % 2)
```

Reply:
Done in [30e42fa](https://github.com/proteneer/timemachine/pull/806/commits/30e42fa1ea70ae75a9506120bdc19a1bb2bc5f6b)